src/garden/Routes.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Redirect, Route, Switch } from 'react-router-dom'
+import Agreement from '../components/Agreement/Agreement'
+import DecisionLoader from '../components/DecisionLoader'
+import Home from '../screens/Home'
+import Profile from '../screens/Profile'
+import ProposalLoader from '../components/ProposalLoader'
+import StakeManagement from '../components/Stake/StakeManagement'
+import { useDAO } from '../providers/Dao'
+
+export default function Routes() {
+  const { connectedDAO } = useDAO()
+  const { address: daoAddress } = connectedDAO || {}
+
+  return (
+    <Switch>
+      <Redirect exact from={`*/${daoAddress}/`} to={`${daoAddress}/home`} />
+      <Route path="*/home" component={Home} />
```

Comment:
As commented in the previous comment, what do you think of updating this to 
```suggestion
      <Route exact path={`*/${daoAddress}/`} component={Home} />
```

So children components that mutates the history, only need to use the current location pathname

Reply:
yeah!